framework/sandstone_p.h

```diff
@@ -470,6 +474,7 @@ struct SandstoneApplication::SharedMemory
     // test execution
     MonotonicTimePoint current_test_endtime = {};
     int current_max_loop_count = 0;
+    std::chrono::duration<int, std::micro> current_test_sleep_duration = 0us;
```

Comment:
Can you do with milliseconds here? If so, we have `ShortDuration`

Please replace the `0us` with `{}`, so you don't need the `using namespace` above.

Reply:
> Can you do with milliseconds here? If so, we have `ShortDuration`

Sadly, no. With milliseconds we don't get enough control over C-state residency.

Reply:
Ok, understood.